addons/website_sale_wishlist/views/website_sale_wishlist_template.xml

```diff
@@ -32,14 +32,15 @@
         </xpath>
     </template>
 
-    <template id="header" inherit_id="website.layout" name="Header Shop Wishlist Link">
-        <xpath expr="//header//ul[@id='top_menu']/li[contains(@t-attf-class, 'o_wsale_my_cart')]" position="after">
+    <template id="header_wishlist_link" inherit_id="website.placeholder_header_wishlist" name="Header Wishlist Link">
```

Comment:
Same as cart:
- placeholder_header_wishlist should be used by this commit
- enabling this commit will indeed enable the wishlist when empty... but if not enabled the wishlist will probably never appear even if there is something inside

Reply:
• Okay. Same as the comment about Navbar Brand.
• You're right, it's buggy. It seems that now, we just want to show or hide this component, whatever it's empty or not. Same for cart.

Reply:
Do you mean we want that now? @sbuhl ?

Reply:
I probably misunderstood feedback from Fabien about this. My bad. I will roll back to the current behavior.